002-routing-params-views/src/app/components/dashboard/config/config.component.js

```diff
@@ -0,0 +1,35 @@
+(function(){
+    'use strict';
+
+    var configComp = {
+        bindings:{
+            configuration: '<'
+        },
+        templateUrl: 'app/components/dashboard/config/config.html',
+        controller: 'ConfigController',
+        controllerAs: 'configCtrl'
+    }
+
+    angular
+        .module('components.dashboard')
+        .component('config', configComp)
+        .config(configConfig);
+
+    /** @ngInject */
+    function configConfig($stateProvider){
+        $stateProvider
+            .state('config', {
+                parent: 'dashboard',
+                url: '/config',
+                component: 'config',
+                onEnter: function(Global){
+                    Global.setTitle(['Dashboard', 'Config'])
```

Comment:
this is used in several route files and is likely to be used further down the road, so for this make an interceptor (remember DRY)